app/api/v1/mediator/election.py

```diff
@@ -114,7 +128,57 @@ def create_election(
     return set_election(election, request.app.state.settings)
 
 
-@router.post("/find", response_model=ElectionQueryResponse, tags=[ELECTION])
+def to_election_summary(election: Election):
+    dto = ElectionSummaryDto(
+        election_id=election.election_id,
+        name=election.get_name(),
+        state=election.state,
+        number_of_guardians=election.context.number_of_guardians,
+        quorum=election.context.quorum,
+        cast_ballots=0,
+        spoiled_ballots=0,
+        index=0,
+    )
+    return dto
+
+
+@router.post(
+    "/list",
+    response_model=ElectionListResponseDto,
+    dependencies=[ScopedTo([UserScope.admin])],
+    tags=[ELECTION],
+)
+def list_elections(
+    request: Request,
+) -> ElectionListResponseDto:
+    """
+    List all elections including state and number of ballots cast or submitted if any.
+    """
+
+    result = ElectionListResponseDto()
+    elections = filter_elections(filter={}, settings=request.app.state.settings)
+    result.elections = [to_election_summary(e) for e in elections]
+    index = 0
+    for election in result.elections:
+        election.index = index
+        inventory = get_ballot_inventory(
```

Comment:
This is an N+1 Selects problem, but there should be very few elections in the system, so I'm inclined to be lazy here unless anyone objects